airgun/widgets.py

```diff
@@ -2685,6 +2696,34 @@ def read(self):
         return self.column_value.read()
 
 
+class FieldWithEditBtn(Widget):
+    """A pair of a field and a button that makes the field editable.
+    After editing, confirm by checkmark or cancel by X.
+    Is present e.g. in PF5 Settings.
+    """
+
+    ROOT = '//td[2]'
+    textinput = TextInput(locator=".//input[@data-ouia-component-type='PF5/TextInput']")
+    textarea = TextInput(locator=".//textarea")
+    dropdown = FormSelect(locator=".//select[@data-ouia-component-type='PF5/FormSelect']")
+    editBtn = PF5Button(locator=".//button[contains(@data-ouia-component-id, 'edit-row')]")
+    confirmBtn = PF5Button(locator=".//button[@data-ouia-component-id='submit-edit-btn']")
+    cancelBtn = PF5Button(locator=".//button[@data-ouia-component-id='cancel-edit-btn']")
```

Comment:
Please use this 
`from widgetastic_patternfly5.ouia import Button as PF5OUIAButton`
```suggestion
    editBtn = PF5OUIAButton('edit-row')
    confirmBtn = PF5OUIAButton('submit-edit-btn')
    cancelBtn = PF5OUIAButton('cancel-edit-btn')
```

Reply:
Partially done but unable for editBtn where `data-ouia-component-id="edit-row-X-icon"` for X being the line number.